### PR TITLE
Feat/web/profile

### PR DESCRIPTION
**Pull request by yuriserka**

### CODE REVIEW

File: web/src/store/metamask/metamask.jsx [hunk 1]
```diff
@@ -173,9 +197,13 @@ export default function MetamaskProvider({ children }) {
...
   async function mint() {
     try {
-      const contentId = 'QmWoAd19aCmkax45YsaH1exYKHk5erv81fMRdMPxgGKmpz';
+      const contentId = "QmWoAd19aCmkax45YsaH1exYKHk5erv81fMRdMPxgGKmpz";
       const tokenId = `https://gateway.pinata.cloud/ipfs/${contentId}/samambaia01.png`;
-      await state.contracts.smartCityGames.connect(state.signer).payToMint(state.account, tokenId, { value: ethers.utils.parseEther('0.5') });
+      await state.contracts.smartCityGames
+        .connect(state.signer)
+        .payToMint(state.account, tokenId, {
+          value: ethers.utils.parseEther("0.5"),
```
Comment (henrique-mariano) [inline web/src/store/metamask/metamask.jsx:205]: Don't forget to change to 0.05

File: web/src/store/metamask/metamask.jsx [hunk 2]
```diff
@@ -173,9 +197,13 @@ export default function MetamaskProvider({ children }) {
 
   async function mint() {
     try {
-      const contentId = 'QmWoAd19aCmkax45YsaH1exYKHk5erv81fMRdMPxgGKmpz';
+      const contentId = "QmWoAd19aCmkax45YsaH1exYKHk5erv81fMRdMPxgGKmpz";
       const tokenId = `https://gateway.pinata.cloud/ipfs/${contentId}/samambaia01.png`;
```
Comment (henrique-mariano) [inline web/src/store/metamask/metamask.jsx:201]: In the future makes more sense to store only the CID of the file

**Review by henrique-mariano — APPROVED** [review]

File: web/src/store/metamask/metamask.jsx (hunk 2 above)
Comment (yuriserka) [reply web/src/store/metamask/metamask.jsx]: I'm still confused how this is going to work

Comment (henrique-mariano) [reply web/src/store/metamask/metamask.jsx]: I will explain better

**Pull request merged**